config/default.yml

```diff
@@ -114,7 +114,7 @@ Metrics/BlockLength:
   Enabled: true
   VersionAdded: 'TBD'
   Exclude:
-    - 'spec/**/_spec.rb'
+    - 'spec/**/*_spec.rb'
```

Comment:
Maybe follow the same naming scheme as a common `Include` one above?
```
- "**/*_spec.rb"
- "**/spec/**/*"
```

This has one additional benefit - shared examples/contexts (e.g. `spec/support/shared_examples/it_behaves_like_everyone.rb`) would be excluded, too, even though they typically don't end with `_spec.rb`.